packages/kestrel_core/src/kestrel/ir/instructions.py

```diff
@@ -33,6 +35,21 @@
     InitVar.__call__ = lambda *args: None
 
 
+class MashDataFrame(DataFrame, SerializableType):
```

Comment:
Not sure what `MashDataFrame` means.  If it's just a serializable DataFrame, maybe call it `SerializableDataFrame`?

Reply:
Good idea. Will rename